### DISCUSSION TITLE
npc Packet PlayerPublicKey

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#1389] [Help] [answered]
Hello, a few discussions ago I have asked how I could create a npc with a skin. The code was: 
```
addPlayerPacket = PlayerInfoPacket(PlayerInfoPacket.Action.ADD_PLAYER, PlayerInfoPacket.AddPlayer(uuid, displayName, listOf(PlayerInfoPacket.AddPlayer.Property("textures", skin.textures(), skin.signature())), GameMode.CREATIVE, 0, Component.text(displayName)))
```
But now I have updated to the latest version, and minestom expect a PlayerPublicKey for the constructor. I have a few questions about that.
1:  What is a PlayerPublicKey
2:  How can I implement it?

**Comment by emortaldev** [comment, marked as answer]
You can just put null
